### PR TITLE
698 install packages prior to attempting to build their documentation

### PR DESCRIPTION
**Pull request by jsolbrig**
# Reviewer Checklist

PR author: Please ensure you meet all of the below requirements, and check boxes appropriately.

Reviewers: Please confirm all required testing/documentation has been completed prior to approving.

**Remove lines that are not applicable, explain if you select "NO REQUIRED"**

* [ ] Required ***existing tests*** pass (ie full_test.sh, others as appropriate)
* [ ] NO REQUIRED ***existing tests*** (explain why not required)
* [ ] Required ***unit tests*** added and pass for new/modified functionality
* [ ] NO REQUIRED ***unit tests*** (explain why not required)
* [ ] Required ***integration tests*** added and pass for new/modified functionality
* [ ] NO REQUIRED ***integration tests*** (explain why not required)
* [ ] Required ***documentation*** added for new/modified functionality
* [ ] NO REQUIRED ***documentation*** (explain why not required)
* [ ] Required ***release notes*** added for new/modified functionality
* [ ] NO REQUIRED ***release notes*** (explain why not required)
* [ ] Required ***updates to other repos*** complete
* [ ] NO REQUIRED ***updates to other repos*** (explain why not required)

https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#NNN

# Testing Instructions

# Summary

# Output

### CODE REVIEW

File: docs/source/releases/latest/698-install-packages-prior-to-attempting-to-build-their-documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    - ''
+    modified:
+    - 'docs/build_docs.sh'
+    - '.github/workflows/doc-lint-test.yaml'
+    moved:
+    - ''
+  related-issue:
+    number: 0
+    repo_url: ''
+  title: 'Install packages prior to attempting to build their documentation'
```
Comment (mindyls) [inline docs/source/releases/latest/698-install-packages-prior-to-attempting-to-build-their-documentation.yaml:22]: @biosafetylvl5  Does brassy ignore all the empty ones?  Or do we end up with a bunch of empty sections in latest.rst?

**Review by mindyls — APPROVED** [review]
Looks good - note my question about he empty sections in YAML release notes.  Mostly I just want to know, I don't care that much for this PR.

File: docs/source/releases/latest/698-install-packages-prior-to-attempting-to-build-their-documentation.yaml (hunk 1 above)
Comment (mindyls) [reply docs/source/releases/latest/698-install-packages-prior-to-attempting-to-build-their-documentation.yaml:22]: Jeremy said it was fine :)

**Review by evrose54 — APPROVED** [review]

File: docs/source/releases/latest/698-install-packages-prior-to-attempting-to-build-their-documentation.yaml (hunk 1 above)
Comment (biosafetylvl5) [reply docs/source/releases/latest/698-install-packages-prior-to-attempting-to-build-their-documentation.yaml:22]: It should!

**Review by biosafetylvl5 — APPROVED** [review]
LGTM!

**Pull request merged**